### ISSUE TITLE
Minimized code throws a type error

### ISSUE THREAD

**Issue by flekschas** [mattslocum/ng-webworker#5]
I tried to run the simple example using the minimized version and it throws the following error 

> Uncaught SyntaxError: Unexpected token (

This error points to the following part of the worker code (see the arrow):

``` javascript
var _async_ = false;function (a){var b=[];
                             ↑
```

The original version works fine.

**Edit**:
Looks like `_transferable_()` get's screwed by Uglify.

**Comment by mattslocum** [comment]
Thanks @flekschas! Merged

**Comment by chmiiller** [comment]
I know that this issue has been solved when #6 was merged but we've been having it using grunt-contrib-uglify version 4.x.
It seems that the `_transferable_` function name is compressed because the function is used only once so our fix was to set compress to: `{keep_fnames: true}`, like this :
```uglify: {
            my_target: {
                options: {
                    compress: {
                        keep_fnames: true
                    },
                    mangle: {
                        except: ['notify', 'complete', '_transferable_']
                    },
                    preserveComments: "some"
                },
                files: {
                    'src/ng-webworker.min.js': ['src/ng-webworker.js'],
                    'src/worker_wrapper.min.js': ['src/worker_wrapper.js']
                }
            }
        }
```
on the Gruntfile.js
